### ISSUE TITLE
r.rterm.* cannot support Variables Reference

### ISSUE THREAD

**Issue by JianhuiShi** [REditorSupport/vscode-R#703]
System: Windows 10. 
VSCode: Version 1.5.8.
vscode-R: v 2.0.0 release.

This is my current settings. I create a R terminal by `R: Create R terminal` and everything works fine.
```json
    "r.rterm.windows": "D:\\Miniconda3\\Scripts\\radian.exe",
    "r.bracketedPaste": true,
    "r.rterm.option": [
        "--r-binary=D:\\Program Files\\R\\R-3.6.3\\bin\\x64\\R.exe"
    ],
    "r.sessionWatcher": true,
    "r.alwaysUseActiveTerminal": true,
    "r.rpath.windows": "D:\\Program Files\\R\\R-3.6.3\\bin\\x64\\R.exe"
```
But this settings cannot work on another PC, on which R is installed in `C:\Program Files\R\R-3.6.3 `and radian is installed in `C:\Miniconda3\Scripts`.
So in the settings I try to substitute "D:" with Environment Variable "SystemDrive" using Variables Reference, see below:
```json
    "r.rterm.windows": "${env:SystemDrive}\\Miniconda3\\Scripts\\radian.exe",
    "r.bracketedPaste": true,
    "r.rterm.option": [
        "--r-binary=${env:SystemDrive}\\Program Files\\R\\R-3.6.3\\bin\\x64\\R.exe"
    ],
    "r.sessionWatcher": true,
    "r.alwaysUseActiveTerminal": true,
    "r.rpath.windows": "${env:SystemDrive}\\Program Files\\R\\R-3.6.3\\bin\\x64\\R.exe"
```
But it doesn't work. When I try `R: Create R terminal` , error happens:
```
Cannot find R client at ${env:SystemDrive}\Miniconda3\Scripts\radian.exe. Please check r.rterm setting.
```
Can r.rterm.* support Variables Reference and how can I do?
Thanks!

**Comment by renkun-ken** [comment]
The variable substitution is not supported yet. It is perfectly possible to implement it.

**Comment by baslat** [comment]
> The variable substitution is not supported yet. It is perfectly possible to implement it.

Was this implemented? I'm trying to do something similar but with workspace settings and radian installed in workspace python environments, and cannot get the r.rterm setting to refer to ${workspaceFolder}

**Comment by renkun-ken** [comment]
Related: https://github.com/microsoft/vscode/issues/46471

**Comment by renkun-ken** [comment]
[vscode-variables](https://www.npmjs.com/package/vscode-variables) suggeted at https://github.com/microsoft/vscode/issues/46471#issuecomment-762773662 could be useful.

**Comment by renkun-ken** [comment]
Resolved in #1444
